packages/react-admin-stories/.storybook/config.tsx

```diff
@@ -9,7 +11,19 @@ function loadStories() {
 }
 
 addDecorator(story => {
-    const theme = createMuiTheme({});
+    const theme = createMuiTheme({
```

Comment:
Im Storybook Theme sollte find ich nichts gemacht werden, man sollte ehr den "blanken" Zustand der Komponenten sehen können. Also, dass das gleich ausschaut, wenn ich das in einem neuen Projekt verwende. 

Wenn das Styling sonst nicht passt, besser das Default-Styling anpassen.

Reply:
@jamesricky da gehts eigentlich nur darum, dass das input-field visuell sichtbar ist. Ich wollte die styles nicht in das allgemeine theme schreiben weil das dann ja fürs ganze storybook greift... Die default-styles kommen von InputBase dafür ist aber in der story-book config nichts gesetzt...